resources/public/api/conf/1.0/schemas/retrievePeriodicObligations/retrieve_periodic_obligations.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve obligations for a user's business income sources",
+  "description": "An array to return one or more businesses and their obligations details",
+  "type": "object",
+  "properties": {
+    "obligations": {
+      "type": "array",
+      "description": "An array to return one or more businesses and their obligations details",
+      "items": {
+        "type": "object",
+        "properties": {
+          "typeOfBusiness": {
+            "description": "Identifies the type of business income source. One of \"self-employment\", \"uk-property\" or \"foreign-property\"",
+            "type": "string",
+            "example": "self-employment"
+          },
+          "businessId": {
+            "description": "The unique identifier for this business income source",
+            "type": "string",
+            "example": "XAIS12345678910"
+          },
+          "obligationDetails": {
+            "type": "array",
+            "description": "An array holding the obligations for the business income source",
+            "items": {
+              "type": "object",
+              "properties": {
+                "periodKey": {
```

Comment:
I don't believe we are showing period key to the vendors

Reply:
according to the spec it is. I'll ask Francis in the morning